salt/modules/groupadd.py

```diff
@@ -75,21 +75,24 @@ def add(name, gid=None, system=False, root=None, non_unique=False):
 
         .. versionadded:: 3006.0
 
+    local
+        Specifically add the group locally rather than through remote providers (e.g. LDAP)
```

Comment:
```suggestion
        Specifically add the group locally rather than through remote providers (e.g. LDAP)
        
        .. versionadded:: 3007.0
```